src/components/molecules/ProjectSearchHead/ProjectSearchHead.tsx

```diff
@@ -43,7 +45,12 @@ export const ProjectSearchHead = (props: ProjectSearchHeadProps) => {
           <Tbody borderRight="1px solid #B8BEC0">
             {projectResults?.data ? (
               projectResults.data.map((projectResults, idx) => (
```

Comment:
do the map outside of rendering then use the resulting array in rendering instead.